application/features/Audio.py

```diff
@@ -57,13 +60,16 @@ def __del__(self):
         super().__del__()
 
     def connect(self, *args, **kwargs):
+        logger.debug("Audio: Establishing Audio connection")
         return super().connect(*args, **kwargs)
 
     def launch_audio(self):
         try:
+            logger.debug("Audio: Launching Audio connection. Forwarding request to 127.0.0.1, port 0")
```

Comment:
i think we can omit the address since that's hardcoded in the code.
```suggestion
            logger.debug("Audio: Launching Audio connection.")
```

btw, by specifying `port=0` we are actually requesting "port=any" allocated by the OS.

Reply:
Thanks for the info!